ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManagerTest.java

```diff
@@ -1124,9 +1124,9 @@ public void forkIdForChainHeadMayBeNull() {
 
   @Test
   public void shouldUseRightCapabilityDependingOnSyncMode() {
-    assertHighestCapability(SyncMode.X_SNAP, EthProtocol.ETH67);
-    assertHighestCapability(SyncMode.FULL, EthProtocol.ETH67);
-    assertHighestCapability(SyncMode.X_CHECKPOINT, EthProtocol.ETH67);
+    assertHighestCapability(SyncMode.X_SNAP, EthProtocol.ETH68);
+    assertHighestCapability(SyncMode.FULL, EthProtocol.ETH68);
+    assertHighestCapability(SyncMode.X_CHECKPOINT, EthProtocol.ETH68);
     /* Eth67 does not support fast sync, see EIP-4938 */
     assertHighestCapability(SyncMode.FAST, EthProtocol.ETH66);
```

Comment:
instead of replacing eth/67 with eth/68 could you test both?

Reply:
So that test asserts the highest capability available for a sync mode. If 68 is enabled, then it will always be 68. Does it make sense? Each Sync mode will have only one possible highest capability

Reply:
ok, I missed the _highest capability_ part, probably